### PR TITLE
fix: add prefixes to Redis

### PR DESCRIPTION
**Pull request by Atala**
F*x #63 

I think we just need to redeploy each instance and it will be fine, as data are always polled from Postgres.

Note that I did a small trick to prefix `subscribe` calls.

<strike>Please don't merge I have still a [question about this change](https://github.com/NodeRedis/node_redis/issues/1286)</strike>

### CODE REVIEW

**Comment by Atala** [comment]
I don't know what happened with the `fsevents` dependency, is it used for something?

**Comment by alexsegura** [comment]
I'll check this week.

File: js/app/cart/Cart.jsx [hunk 1]
```diff
@@ -101,7 +101,6 @@ class Cart extends React.Component
             }
 
         $.post(this.props.addToCartURL, {
-          date: this.props.deliveryDate,
```
Comment (alexsegura) [inline js/app/cart/Cart.jsx]: Is this normal?

Comment (Atala) [reply js/app/cart/Cart.jsx]: Yep (I dont remember why this change is here though)

Comment (alexsegura) [reply js/app/cart/Cart.jsx]: I extracted it into a single commit 😉

**Pull request merged**